doc/JSON-RPC-interface.md

```diff
@@ -5,6 +5,79 @@ The headless daemon `bitcoind` has the JSON-RPC API enabled by default, the GUI
 option. In the GUI it is possible to execute RPC methods in the Debug Console
 Dialog.
 
+## Security
+
+The RPC interface allows other programs to control Bitcoin Core,
+including the ability to spend funds from your wallets, affect consensus
+verification, read private data, and otherwise perform operations that
+can cause loss of money, data, or privacy.  This section suggests how
+you should use and configure Bitcoin Core to reduce the risk that its
+RPC interface will be abused.
+
+Anyone with physical or remote access to the computer, container, or
+virtual machine running Bitcoin Core can compromise either the whole
+program or just the RPC interface.  This includes being able to record
+any passphrases you enter for unlocking your encrypted wallets or
+changing settings so that your Bitcoin Core program tells you that
+certain transactions have multiple confirmations even when they aren't
+part of the best block chain.  For this reason, you should not use
+Bitcoin Core for security sensitive operations on systems you do not
+exclusively control, such as shared computers or virtual private
+servers.
+
+By default, the RPC interface can only be accessed by a client running
+on the same computer and only after the client provides a valid
+authentication credential (username and passphrase).  Any program on
+your computer with access to the file system and local network can
+obtain this level of access.  Additionally, other programs on your
+computer can attempt to provide an RPC interface on the same port as
+used by Bitcoin Core in order to trick you into revealing your
+authentication credentials.  For this reason, it is important to only
+use Bitcoin Core for security-sensitive operations on a computer whose
+other programs you trust.
+
+You may optionally allow other computers to remotely control Bitcoin
+Core by setting the `rpcallowip` and `rpcbind` configuration parameters.
+**Do not enable RPC connections over the public Internet.**  These
```

Comment:
Don't know if it is necessary but could mention ssh forwarding of the RPC port, as an option that does provide encryption and and can be used over the internet.